portal/portal-sprint-101.md

```diff
@@ -0,0 +1,38 @@
+---
+slug: /release-notes/portal/sprint-101
+title: Sprint 101 (May 31, 2021)
+---
+
+# fiskaltrust.Portal - Sprint 101
+_May 31, 2021_
+
+**Usability improvements**
+
+In this sprint, we have introduced several usability improvements including improving the overall performance.
+
+## Features
+
```

Comment:
Should we also mention this Pbi in the release notes?
#42260 Integrate basic read tracking into KBA visualization